nuitka/nodes/ImportNodes.py

```diff
@@ -1002,11 +1012,12 @@ def _attemptFollow(self, module_name):
 
                     if name_import_module_filename is not None:
                         self.used_modules.append(
-                            (
+                            ModuleUsageAttempt(
                                 name_import_module_name,
                                 name_import_module_filename,
                                 name_import_finding,
                                 1,
+                                None,
```

Comment:
We would have a `source_ref` here, do we not?

Reply:
yes